docs/02-developers/04-quickstart/remix.md

```diff
@@ -0,0 +1,180 @@
+
+![Deploy and Run Transactions](/img/developers/quickstart/6-remix.png)
+
+Confirm the transaction in MetaMask:
+
+![Confirm transaction deployment in MetaMask](/img/developers/quickstart/7-remix.png)
+
+If deployment is successful, it will appear under **Deployed/Unpinned Contracts** section
+
+![View deployed/unpinned contract](/img/developers/quickstart/8-remix.png)
+
+## Interacting with the Contract
+
+As shown in the previous step after deploying the contract, you can expand the desired contract and interact with it, calling the available functions by entering the required arguments, if any. Once entered the arguments, click on the method name and, in the event that it’s a writing method, Metamask will prompt you to sign the transaction. If the method is read only, it will show the return value under the method name.
+
+## Verifying the Contract on Rootstock Explorer
+
+Go back to Remix file explorer and go to `contracts -> artifacts -> build-info`. In this folder there will be a json file containing the information of the compilation process. You’ll need this in order to verify the contract on the [Rootstock Testnet Explorer](https://explorer.testnet.rootstock.io/). Copy the whole content of the input attribute (just the value; what’s inside the curly brackets) and paste it in a separate json file in your computer or code editor.
+
+![deployed contract artifact](/img/developers/quickstart/9-remix.png)
+
+Copy the contract's address from the Remix deployment output, you can find the contract address under the **Deployed/Unpinned** Contracts.
+
+![Copy contract address](/img/developers/quickstart/10-remix.png)
+
+Visit the [Rootstock Testnet Explorer](https://explorer.testnet.rootstock.io/) and paste the Contract Address in the search field.
+
+![Paste Contract Address - Rootstock Testnet Explorer](/img/developers/quickstart/11-remix.png)
+
+Click on the Code tab and click the button to **Verify Contract**.
+
+![View Contract Address - Rootstock Testnet Explorer](/img/developers/quickstart/12-remix.png)
+
+Select **Standard JSON Input** as the verification method and fill the form:
+
+1. Standard JSON input: In the standard JSON input, upload the file you created from the build info of the contract.
+2. Contract name: The name of the contract which you declared it with. Remember that it’s important the file of the contract has the same matching name as the standard-json-input.json. Now, click on add file to upload.
+3. Compiler: This is the compiler version the contract has been compiled with. For example 
+Constructor arguments: The constructor arguments of the contract, if any. If you don’t know the arguments at first, continue with the process and the explorer will suggest the ones to use.
+4. ABI encoded arguments: if the arguments provided are ABI encoded, check this option.
+
+> Copy only the information inside the input code block {}, paste the code into a file named `standard-json-input.json` and click on add file:
```

Comment:
Same thing from the previous comment. Make sure user understands that he needs to paste the whole input value. This means curly braces included because the value is a details object { }